docs/notebooks/06_differential_expression.ipynb

```diff
@@ -0,0 +1,1306 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #15.        sp.run(deseq_cmd, capture_output=False, stdout=stdout, stderr=stderr)
</pre><p>I would add <code>check=True</code> ... otherwise if there's an error it will just fail silently.  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/icbi-lab/atlas_protocol/pull/13/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>